Dockerfile

```diff
@@ -25,7 +25,6 @@ COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifi
 COPY --from=builder /go/src/github.com/pusher/oauth2_proxy/oauth2_proxy /bin/oauth2_proxy
 COPY --from=builder /go/src/github.com/pusher/oauth2_proxy/jwt_signing_key.pem /etc/ssl/private/jwt_signing_key.pem
 
-RUN addgroup -S -g 2000 oauth2proxy && adduser -S -u 2000 oauth2proxy -G oauth2proxy
-USER oauth2proxy
+USER 2000
```

Comment:
Using `USER` without a group ID will run the process GID `root` which isn't desirable:
```
$ docker run -ti --entrypoint /bin/ash quay.io/pusher/oauth2_proxy:kskewes-user
/ $ id
uid=2000 gid=0(root)
```

You could specify a generic GID as well, though, like `USER 2000:2000` or something like that.

Reply:
Thanks, good point, fixed.

Reply:
```
karl@k8s-w-01:~$ docker pull docker.io/karlskewes/oauth2_proxy:latest-arm64 
latest-arm64: Pulling from karlskewes/oauth2_proxy
5011838a0b2d: Already exists 
6ca3e9f5b84e: Already exists 
b934274890d1: Pull complete 
f1557abb7fff: Pull complete 
Digest: sha256:9de4246e8832178d2dc354df09a8cef4cd47bf19f9862595c39499515c2e9343
Status: Downloaded newer image for karlskewes/oauth2_proxy:latest-arm64
karl@k8s-w-01:~$ docker run --rm -it --entrypoint ash docker.io/karlskewes/oauth2_proxy:latest-arm64 
/ $ id
uid=2000 gid=2000
```